rdr_service/dao/genomics_dao.py

```diff
@@ -3040,6 +3040,46 @@ def get_report_state_from_wf_state(wf_state):
                 return value
         return None
 
+    def get_hdr_result_positive_no_appointment(self) -> List[str]:
+        """Returns participants with hdr positive result over 14 days ago and no appointment scheduled/completed"""
+        result_before = clock.CLOCK.now() - timedelta(days=14)
```

Comment:
Can we add this as a param with a default?
`get_hdr_result_positive_no_appointment(self, days=14)`
`result_before = clock.CLOCK.now() - timedelta(days)`